src/imputation/imputation_main.py

```diff
@@ -50,6 +50,15 @@ def run_imputation(
     Returns:
         pd.DataFrame: dataframe with the imputed columns updated
     """
+    # add osmotherly & key businesses columns to PNP data
+    if config["survey"]["survey_type"] == "PNP":
+        df = hlp.identify_key_business(df)
+        df = hlp.identify_osmotherly_businesses(df)
+        df = hlp.add_area_column(df)
+        df = hlp.create_imp_class_col(df,
+                                  ["pnp_key", "area"],
+                                  use_osmotherly=True,
+                                  use_cellno=False)
```

Comment:
If we create the imputation class here, we'll get stuck when the MoR code tries to do it again- so maybe we could move this code to where the create_imp_class_col is created in the MoR script.